[Fix] MapStruct를 사용할 때 Bean 주입 에러 해결

### 🐛 버그 내용
```
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'fundingController' defined in file 

Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'fundingService' defined in file

Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'com.ssafy.funding.mapper.FundingMapper' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {}
```

--- a/back/funding/src/main/java/com/ssafy/funding/mapper/FundingMapper.java
+++ b/back/funding/src/main/java/com/ssafy/funding/mapper/FundingMapper.java
@@ -5,7 +5,7 @@ import com.ssafy.funding.domain.Funding;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
 
-@Mapper
+@Mapper(componentModel = "spring")
 public interface FundingMapper {
     @Mapping(source = "funding.time", target = "startTime")
     @Mapping(source = "count", target = "count")

